dashboard/config/foorm/forms/surveys/teachers/teacher_end_of_year_survey.json

```diff
@@ -0,0 +1,1060 @@
+      },
+      {
+       "value": "This_curriculum_was_effec",
+       "text": "This curriculum was effective for my students."
+      },
+      {
+       "value": "The_teaching_strategies_r",
+       "text": "The teaching strategies recommended in the curriculum and lesson plans were clear to me."
+      },
+      {
+       "value": "The_tools_and_technologie",
+       "text": "The tools and technologies that are part of the curriculum make sense and are easy to use."
+      }
+     ]
+    },
+    {
+     "type": "matrix",
+     "name": "challenges_remote",
+     "title": "Please characterize the challenges you faced teaching your students with Code.org remotely. ",
+     "columns": [
+      {
+       "value": "1",
+       "text": "Not a challenge"
+      },
+      {
+       "value": "2",
+       "text": "Some challenges"
+      },
+      {
+       "value": "3",
+       "text": "Huge challenge"
+      }
+     ],
+     "rows": [
+      {
+       "value": "Internet_and_device_acces",
+       "text": "Internet and device access for me or my students"
+      },
+      {
+       "value": "Providing_support_and_ass",
+       "text": "Providing support and assistance while students work"
+      },
+      {
+       "value": "Tracking_student_progress",
+       "text": "Tracking student progress on assigned materials"
+      },
+      {
+       "value": "Providing_feedback_on_wor",
+       "text": "Providing feedback on work students have done"
+      },
+      {
+       "value": "Assigning_new_work",
+       "text": "Assigning new work"
+      },
+      {
+       "value": "Finding_materials_that_st",
+       "text": "Finding materials that students can work on independently"
+      },
+      {
+       "value": "Getting_students_their_lo",
+       "text": "Getting students their login information"
+      },
+      {
+       "value": "Finding_materials_to_work",
+       "text": "Finding materials to work on that don't require internet and device access"
+      },
+      {
+       "value": "Communicating_with_my_stu",
+       "text": "Communicating with my students while they are at home"
+      },
+      {
+       "value": "Providing_instruction_on_",
+       "text": "Providing instruction on new concepts to my students"
+      },
+      {
+       "value": "Finding_ways_for_students",
+       "text": "Finding ways for students to collaborate"
+      }
+     ]
+    },
+    {
+     "type": "comment",
+     "name": "one_challenge_FR_remote",
+     "title": "For one of the items you identified as a huge challenge, please describe any ideas you have for how code.org might address it.",
+     "cols": 40,
+     "rows": 6
+    },
+    {
+     "type": "matrix",
+     "name": "cdo_strategies_remote",
+     "title": "When teaching {course} remotely how often did YOU explicitly do the following?",
+     "columns": [
+      {
+       "value": "1",
+       "text": "Never"
+      },
+      {
+       "value": "2",
+       "text": "Rarely<br>(a few times a year)"
```

Comment:
looks like we can't include html here--will get rendered as plaint text

Reply:
shoot.  I removed those.  must have gotten clobbered in some edit.  will fix